purge_test.go

```diff
@@ -20,7 +19,7 @@ type FastlyPurgeSuite struct {
 }
 
 const (
-	VALID_PURGE_URL               string = "http://www.example.com/test/sample.jpg"
+	VALID_PURGE_PATH              string = "/test/sample.jpg"
```

Comment:
Because I couldn't run a stub HTTP server in the tests for http://www.example.com, so I build the full URL within the test methods now